projects/views.py

```diff
@@ -350,6 +351,33 @@ def post(self, request, project_slug):
             project.authorized.add(selected_user)
             assign_perm("can_view_project", selected_user, project)
 
+            project_uri = f"{request.get_host()}/projects/{project.slug}"
+            email_body = f"""\
```

Comment:
```suggestion
            email_body = f"""
```

Do you need an escape character here?

Reply:
It seems quite obvious to me, maybe adding a comment here is an overkill? I'm happy to add one though if you think we should so let me know :)